google/cloud/logging/logging_service_v2_options.gcpcxx.pb.h

```diff
@@ -30,6 +30,7 @@ namespace cloud {
 namespace logging {
 inline namespace GOOGLE_CLOUD_CPP_GENERATED_NS {
 
+/// Options to use with `google::cloud::Options`.
```

Comment:
Micronit: Options -> Option as the first word.

Reply:
I envisioned this comment applying to all the *Option types defined below it.